setup_icu4c_src.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+# Get ICU source and prepare for `make dist`
+# RELEASE_BRANCH must be set to the desired Github branch of unicode/icu.
+if [ -z "$RELEASE_BRANCH" ]; then
+   echo "RELEASE_BRANCH is not set."
+   exit 1
+fi
+
+echo "Release branch is" $RELEASE_BRANCH
+cd src
+rm -rf icu
+
+git clone --branch $RELEASE_BRANCH --depth 1 https://github.com/unicode-org/icu.git
+cd icu
+git lfs install --local
+git lfs fetch
+git lfs checkout
+
+# Change LICENSE link to actual LICENSE file in icu4c.
```

Comment:
good idea, though it modifies the repo.  Is LICENSE otherwise not readable?   Perhaps the make dist should just treat symlinks-as-regular-files.  If the file isn't readable from within docker, may need to bind-mount the parent dir (i.e. `icu` not `icu/icu4c` )

Reply:
Thanks. I'm happy to have the `make dist` use the symlink, but am not sure where to find that.

Reply:
I'm assuming there was a specific problem that necessitated this? 

it's probably in the `tar` and `zip` calls with make dist.  Perhaps at least make sure there's a ticket for it.